oxidation/client/src/locales/en-US.json

```diff
@@ -7,10 +7,11 @@
             "myDevices": "My devices"
         },
         "organizationList": {
-            "title": "List of your organizations",
+            "title": "Your organizations",
+            "subtitle": "With Parsec, keep full control of your data and be sure that it safe.",
```

Comment:
Better translation:
`Keep full control over your data with Parsec and be sure of its security.`

Reply:
Done